vendor/github.com/golang/protobuf/jsonpb/jsonpb.go

```diff
@@ -182,7 +183,12 @@ func (m *Marshaler) marshalObject(out *errWriter, v proto.Message, indent, typeU
 				return fmt.Errorf("failed to marshal type URL %q to JSON: %v", typeURL, err)
 			}
 			js["@type"] = (*json.RawMessage)(&turl)
-			if b, err = json.Marshal(js); err != nil {
```

Comment:
we usually call they commits `bump(*): kubernetes 1.17-rc.2`

Reply:
@sttts I updated the commit messages, please let me know if you have any further suggestions.